src/common/grpc-expr/src/util.rs

```diff
@@ -151,6 +151,15 @@ pub fn build_create_table_expr(
 
         let column_type = infer_column_datatype(datatype, datatype_extension)?;
 
+        ensure!(
```

Comment:
https://github.com/GreptimeTeam/greptimedb/pull/5422 also forbids the interval type in another place.

Where is the best place to do this validation? We may need to ensure all create table operations use the same validation rule.

Reply:
Looks like we have many different helper method to build a create table expr......